pycoast/tests/test_pycoast.py

```diff
@@ -677,31 +782,59 @@ def test_add_shapefile_shapes_agg(self):
         self.assertTrue(
             fft_metric(grid_data, res), 'Writing of Brazil shapefiles failed')
 
-    @unittest.skip("All kwargs are not supported, so can't create equal results")
+#    @unittest.skip("All kwargs are not supported, so can't create equal results")
     def test_config_file_coasts_and_grid(self):
         from pycoast import ContourWriterAGG
         from pyresample.geometry import AreaDefinition
         overlay_config = os.path.join(os.path.dirname(__file__),
                                       "coasts_and_grid_agg.ini")
         grid_img = Image.open(os.path.join(os.path.dirname(__file__),
-                                           'grid_nh_agg.png'))
+                                           'grid_nh_cfg_agg.png'))
```

Comment:
Does this mean this test never actually worked? Why doesn't this test need to be skipped now?

Reply:
There was a bug in the .ini file for that unit test, which cause an exception without any image been generated. I think that is why it has had been skipped. I fixed the .ini file and the test can generate image from config file but I need to generate a new source image to compare with.